src/System.Runtime.InteropServices.RuntimeInformation/tests/DescriptionNameTests.cs

```diff
@@ -11,7 +12,8 @@ public class DescriptionNameTests
         [Fact]
         public void VerifyRuntimeDebugName()
         {
-            Assert.Equal(".NET Core", RuntimeInformation.FrameworkDescription);
+            string expected = string.Format(".NET Core {0}", typeof(object).GetTypeInfo().Assembly.GetName().Version.ToString());
```

Comment:
Yes eventually when they bring ToF to open, then there will be a mechanism to separate .NETNative, Desktop and .NETCore tests. Right now, the .NET Native tests are internal, and I will change those tests, after these changes get mirrored.